cosmwasm/packages/accounting/src/contract.rs

```diff
@@ -0,0 +1,1161 @@
+use std::marker::PhantomData;
+
+use anyhow::{bail, ensure, Context};
+use cosmwasm_std::{CustomQuery, Deps, DepsMut, Event, Order, StdResult};
+use cw_storage_plus::Bound;
+use thiserror::Error as ThisError;
+
+use crate::{
+    msg::Instantiate,
+    state::{
+        account::{self, Balance},
+        transfer, Account, Kind, Modification, Transfer, ACCOUNTS, MODIFICATIONS, TRANSFERS,
+    },
+};
+
+/// Instantiate the on-chain state for accounting.  Unlike other methods in this crate,
+/// `instantiate` does not perform any validation of the data in `init`.:w
```

Comment:
This looks like a vim typo (see end of the line). Doesn't matter to me, so your call whether we fix this in this PR or we fix the typo later.

Reply:
Ack.  Will fix in a followup PR.

Reply:
Nvm.  Had to rebase to fix merge conflicts so I fixed this as well.